### ISSUE TITLE
Scroll to active menu item on mobile, doesn't work as it did in v9

### ISSUE THREAD

**Issue by langz** [dnbexperience/eufemia#2134]
When going to docs(subpages) like `/uilib/components/accordion/demos/` (https://deploy-preview-2120--eufemia.netlify.app/uilib/components/accordion/demos/) and then pressing the hamburger/🍔 /sidebar-icon, it doesn't scroll to the active/correct menu item. See the following video:

https://user-images.githubusercontent.com/1359205/226867440-e0452c27-d428-422f-99d2-6c923c95ac65.mov

I would expect that it works as in v9, seen at https://eufemia.dnb.no/ 
So that when visiting the demos tab of accordion and pressing the hamburger-icon, I would expect to be "auto scrolled" to the active item(Accordion) in the menu.
See the following video:

https://user-images.githubusercontent.com/1359205/226867641-e2170988-3562-45fe-ab1e-01ac04f8134a.mov

Not sure if the cause of this is https://github.com/dnbexperience/eufemia/pull/2063 or https://github.com/dnbexperience/eufemia/pull/2120 🤔 
Could be somewhere else as well 😂

**Comment by joakbjerk** [comment]
Yeah I noticed some issues with the scrolling there myself when working on #2063. I did test it on the v10 without the changes from #2063 and there was still some issues with the alignment of the active menu item after scrolling, so that issue at least happened some time after starting on v10, but before #2063. I made a [jira task ](https://jira.tech.dnb.no/secure/RapidBoard.jspa?rapidView=1221&projectKey=EDS&view=detail&selectedIssue=EDS-318) back then

**Comment by langz** [comment]
This now works as expected 🙏
